pages/common/git-config.md

```diff
@@ -12,9 +12,9 @@
 
 `git config --list --global`
 
-- List all configuration entries that have been defined either locally or globally:
+- List only system configuration entries (stored in `/etc/gitconfig`), and show their file location:
```

Comment:
I don't completely understand, so its checking /etc/gitconfig 
then checking what settings it has then printing /etc/gitconfig?

Reply:
I'm trying to keep my example in parallel with the others. Neither of the other two are completely correct; because you can change the location of where git config files are stored. I included the `/etc/gitconfig` example because it is the default, similar to the previous two examples.

See:
https://www.git-scm.com/docs/git-config#Documentation/git-config.txt---global

I acknowledge that `--system --show-origin` is kind of a toy example; `--show-origin` is much more helpful with a broader scope, especially where your git config may have multiple include/includif directives.

Reply:
It goes a little broader than I understood. `--show-origin` will also show config from other locations, such as arguments passed on a command line. I'm basically of the opinion that for most use cases, you want to use `--show-origin` when you're using `git config --list`

>```Augment the output of all queried config options with the origin type (file, standard input, blob, command line) and the actual origin (config file path, ref, or blob id if applicable).```

https://www.git-scm.com/docs/git-config#Documentation/git-config.txt---show-origin